presto-hive/src/main/java/com/facebook/presto/hive/util/SerDeUtils.java

```diff
@@ -298,9 +268,46 @@ private static Timestamp getTimestamp(Object object, TimestampObjectInspector in
         return inspector.getPrimitiveJavaObject(object);
     }
 
-    public enum JsonContext
+    private static Type getSerializerType(ObjectInspector inspector)
```

Comment:
Instead of adding this method, use `HiveType.getType(inspector, typeManager)`. You'll need to have a TypeManager, but it looks like everything that uses `SerDeUtils` has one, so that should be pretty easy